test/GPv2Settlement.test.ts

```diff
@@ -186,6 +187,62 @@ describe("GPv2Settlement", () => {
       expect(outTransfers.length).to.equal(tradeCount);
     });
 
+    it("should revert if the limit price is not respected", async () => {
+      const encoder = new SettlementEncoder(testDomain);
+      await encoder.signEncodeTrade(
+        {
+          ...partialOrder,
+          kind: OrderKind.SELL,
+          partiallyFillable: false,
+        },
+        0,
+        traders[0],
+        SigningScheme.TYPED_DATA,
+      );
+
+      await expect(
+        settlement.computeTradeExecutionsTest(
+          encoder.tokens,
+          encoder.clearingPrices({
+            [sellToken]: 1,
+            // NOTE: The price of the buy token is way too high!
```

Comment:
Kind of related to Felix's comment: can you assert before this `expect` that the order limit price as defined in `partialOrder` is indeed "worse" than the price defined by `(1, 1000)`? So the purpose of the test is clearer without this comment. You could also do the same in the other tests.

Reply:
Let me know if its better now. I even set the buy and sell amounts specifically for the unit test (since their values matter for understanding the test).